### PR TITLE
feat: added products page

### PR DESCRIPTION
**Pull request by pradipgharge**

### CODE REVIEW

File: pages/Products/products.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+/* Sidebar styling */
+.filter-container{
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+    top:60px;
+    left:0;
+    width: 300px;
```
Comment (varun-Tad) [inline pages/Products/products.css:17]: You can use rems here for width because you have used rems in other places as well

**Comment by varun-Tad** [comment]
On the products page when I hover over the buy now button and add to the cart button of the card with the 'Sale' badge, the hover effects are not applying like it is for the rest of the cards. Also, your website looks neat.

**Review by Sahil25061999 — APPROVED** [review]
everything looks good love the consistency throughout the UI. I think minor changes required as suggested by varun, rest of the code looks good.

**Comment by pradipgharge** [comment]
> On the products page when I hover over the buy now button and add to the cart button of the card with the 'Sale' badge, the hover effects are not applying like it is for the rest of the cards and the cursor too is not changing to pointer like it is in the rest of the cards. Also, your website looks very neat.

That is because I have kept pointer-events to none for that card, as it is out of stock card.

File: pages/Products/products.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    border: none;
+}
+
+/* Products container styling */
+.products-container{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin-left: 300px;
```
Comment (mehulsatardekar) [inline pages/Products/products.css:76]: use rems here

**Review by mehulsatardekar — APPROVED** [review]
you are good to merge this branch after changes

**Comment by varun-Tad** [comment]
Oh okay. I didn't notice the out-of-stock text. You can probably darken the out-of-stock text a little. Rest all looks good to merge. @pradipgharge

File: pages/Products/products.css (hunk 2 above)
Comment (pradipgharge) [reply pages/Products/products.css:76]: Done

File: pages/Products/products.css (hunk 1 above)
Comment (pradipgharge) [reply pages/Products/products.css:17]: Done

File: pages/Products/products.html [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+                    <hr>
+                    <div class="filter-subtype">
+                        <h3 class="subtype-heading">Sort by</h3>
+                        <div class="radio-input">
+                            <input id="low-to-high" type="radio" name="sort" value="low-to-high" checked>
+                            <label for="low-to-high">Price: Low to High</label>
+                        </div>
+                        <div class="radio-input">
+                            <input id="high-low-to" type="radio" name="sort" value="high-low-to" >
+                            <label for="high-low-to">Price: High to Low</label>
```
Comment (rahulk31) [inline pages/Products/products.html:139]: You would want to change the id, value and for to `high-to-low` instead.

**Review by rahulk31 — APPROVED** [review]
Overall code structure is good and well readable. Can be merged. 👍

File: pages/Products/products.html (hunk 3 above)
Comment (pradipgharge) [reply pages/Products/products.html:139]: Thanks for pointing out , made the changes.

**Pull request merged**